frontend/src/config/sample_config_from_backend.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "name": "[Demo] XGBoost - Training with Confusion Matrix",
+    "description": "[source code](https://github.com/kubeflow/pipelines/blob/master/samples/core/xgboost_training_cm) [GCP Permission requirements](https://github.com/kubeflow/pipelines/blob/master/samples/core/xgboost_training_cm#requirements). A trainer that does end-to-end distributed training for XGBoost models.",
+    "file": "/samples/core/xgboost_training_cm/xgboost_training_cm.py.yaml"
+  },
+  {
+    "name": "[Demo] TFX - Taxi Tip Prediction Model Trainer",
```

Comment:
so how to sync this new file with the existing sample json file?
Sina just has another PR update the sample title.

Reply:
backend/src/apiserver/config/sample_config.json

Reply:
Updated a little bit, I included error message that tells how to update.
https://github.com/kubeflow/pipelines/pull/3086/files#diff-afdea79a57383ab17c1cbd4013a3723aR13

Reply:
thanks, as long as test can catch up the diff and msg people to well sync them.